src/System.Linq/src/System/Linq/Enumerable.cs

```diff
@@ -4,12 +4,16 @@
 using System;
 using System.Collections;
 using System.Collections.Generic;
+using System.Numerics;
 using System.Threading;
 
 namespace System.Linq
 {
     public static class Enumerable
     {
+
+        private static readonly int SIMD_THRESHOLD = 64;
```

Comment:
Ideally how should I handle?  You don't need vectorization for small
workloads.  In my testing, roughly 64 a good size before you apply
vectorization.   Or is it the code styling?
On Aug 31, 2015 12:13 AM, "Dmitry Turin" <notifications@github.com> wrote:

> In src/System.Linq/src/System/Linq/Enumerable.cs
> <https://github.com/dotnet/corefx/pull/3024#discussion_r38284873>:
>
> >  using System.Threading;
> >
> >  namespace System.Linq
> >  {
> >      public static class Enumerable
> >      {
> > +
> > +        private static readonly int SIMD_THRESHOLD = 64;
>
> SIMD_THRESHOLD
>
> doesn't match guidelines
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/dotnet/corefx/pull/3024/files#r38284873>.
>